### DISCUSSION TITLE
SceneOsgVisualizer

### DISCUSSION THREAD

**Discussion by 123smile456** [omnetpp/omnetpp#987] [Q&A]
I have created a new project by myself, which has been connected with inet framework, but SceneOsgVisualizer module cannot get background photos, while inet framework project can. Why?

**Comment by rhornig** [comment]
Off topic: to get meaningful response from the community:
- Do not hijack other discussions. Create your own discussion thread.
- Provide a **meaningful title**. Try to formulate your question in a single line so other community members will get a general idea about your problem and **may** click on the discussion to see the details. 
- Put some effort into formulating your question and provide enough context that it can be answered. Provide reproducible code example, error messages, where you put your image file, what type of image etc.

Your current question is so generic that only a generic answer can be given. My guess is, that image resources are loaded from the image path, which is usually in the projectroot/images directory, however it is hard to tell as you did not provide a screenshot, a console log, or we don't now if you started the simulation from command line or from the IDE.

**Reply by 123smile456** [reply]
![QQ截图20230601113532](https://github.com/omnetpp/omnetpp/assets/134665002/7f9bfcf0-86da-4c2d-9bd0-3dcfa635de27)
Here's some of my code information

**Reply by 123smile456** [reply]
I put the photos in the images subfile and wrote the corresponding path in the sceneImage. The simulation still couldn't load the background photos. I wrote sceneImage = "showcases/desert" still couldn't

**Comment by 123smile456** [comment]
![QQ截图20230601113532](https://github.com/omnetpp/omnetpp/assets/134665002/e5ec68e8-dcac-43b2-95a9-897469747d2e)

**Comment by Inspirational123** [comment]
Im getting a message stating "*.visualizer.osgVisualizer.sceneVisualizer.sceneMaxY Does not match any module parameters" when I try to run that simulation
